uni/lib/str_util.icn

```diff
@@ -14,6 +14,8 @@
 
 package util
 
+import lang
+
```

Comment:
Why is this import needed here?

Is it OK for packages util and lang to import each other?

Reply:
Because str_util contains calls to makeProc{}, which is defined in lang.  I don't see why two packages can't
import each other - so long as there's no cycle in the actual code execution.  Maybe there's a reason, but I
have sample programs that won't work without this change and work fine with it.

Reply:
I think there is a chicken-and-egg bootstrapping issue with packages importing each other. If both import each other, whichever one you compile first will fail because the other package doesn't exist yet. In practice, one can get it bootstrapped by starting with partial or near-empty packages, but it will require multiple compiles to get all the symbols into the packages in the .gdbm files, and then recompiling, before the imports will be fully correct. Kind of a Mr. Yuk, but viable as long as the makefiles are smart enough to bootstrap.